rnacentral_pipeline/databases/data/utils.py

```diff
@@ -28,6 +28,7 @@
     "Y_RNA": "SO:0000405",
     "antisense_RNA": "SO:0000644",
     "autocatalytically_spliced_intron": "SO:0000588",
+    "circRNA": "SO:0002291",
```

Comment:
This is needed for import to run, but we aren't sure we want circRNA or sgRNA really. They get filtered out before sending to ensembl, but we should review what to do about these for next release

Reply:
Issue #157 is for tracking this